notifications/telegram.js

```diff
@@ -31,6 +35,33 @@ async function initCommands() {
         await telegramBot.sendMessage(chatId, payment_details, { parse_mode: 'HTML' })
         await telegramBot.sendMessage(chatId, payment_address)
     });
+    telegramBot.onText(/\/withdraw (.+)/, async (msg, match) => {
```

Comment:
Small suggestion

- Print the error message and show instructions on an empty command (**/\/withdraw ?(.*)/**).

![Captura de pantalla 2024-01-09 a la(s) 4 16 13 p  m](https://github.com/deezy-inc/sat-hunter/assets/126987350/c921098a-169d-4bd9-95d3-050c75e1e5f6)

```
    telegramBot.onText(/\/withdraw ?(.*)/, async (msg, match) => {
        const chatId = msg.chat.id;

        // Extracting name and amount from the command
        const args = match[1].split(" ");
        if (args.length < 2) {
            await telegramBot.sendMessage(chatId, "Usage: /withdraw [name] [amount]");
            return;
        }

        const name = args[0];
        const amount = parseFloat(args[1]);

        if (isNaN(amount) || amount <= 1) {
            await telegramBot.sendMessage(chatId, "Please enter a valid amount in satoshis.");
            return;
        }

        try {
            // Call your create_withdraw_request function
            const { withdrawal_details } = await create_withdraw_request(name, amount);
            await telegramBot.sendMessage(chatId, withdrawal_details, { parse_mode: "HTML" });
        } catch (error) {
            console.error("Error processing withdrawal request:", error);
            await telegramBot.sendMessage(chatId, `An error occurred while processing your request. ${error.message}.`);
        }
    });
```

Reply:
great idea, thx! added